movement_primitives/utils.py

```diff
@@ -0,0 +1,11 @@
+import numpy as np
+
+
+def ensure_1d_array(value, dim, label):
```

Comment:
* For consistency, could you rename `dim` to `n_dims`?
* ... and label reminds me of classification, maybe something like `var_name`?